.github/workflows/update-as.yml

```diff
@@ -29,11 +29,12 @@ jobs:
     - name: Get new A-S tag to be used in the PR info
       run: |
         cd test-fixtures/
+        chmod u+x read_tag_test.sh
         echo "version=$(./read_as_tag.sh)" >> $GITHUB_ENV
     - name: Remove temp file created to store the tag info
       run: |
         cd test-fixtures/
-        rm newest_tag.txt
+        [ ! -e newest_tag.txt ] || rm newest_tag.txt
```

Comment:
This worked in my local tests but not sure if that will work here or if it may be better to create an script with an if sentece so that it does not error if the file does not exists..

Reply:
This will remove ```newest_tag.txt``` if it exists and exit with ```0```. If the file does not exist, it will still exit with ```0```.